storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -6199,8 +6198,7 @@ static bool xtrabackup_replace_log_file(const char *redo_path,
   memset(log_buf + LOG_HEADER_CREATOR, ' ', 4);
 
   /* restore original log format */
-  mach_write_to_4(log_buf + LOG_HEADER_FORMAT,
```

Comment:
if we are not changing format in header, do we need to restore at all?

Reply:
@satya-bodapati in fact we are writing the new format on log header at xtrabackup_init_temp_log:

```
  mach_write_to_4(log_buf + LOG_HEADER_FORMAT, to_int(log_format));

  /* write start_lsn header */
  start_lsn = ut_uint64_align_down(max_lsn, OS_FILE_LOG_BLOCK_SIZE);
  mach_write_to_8(log_buf + LOG_HEADER_START_LSN, start_lsn);

  update_log_temp_checkpoint(log_buf, max_lsn);

  success = os_file_write(write_request, src_path, src_file, log_buf, 0,
                          LOG_FILE_HDR_SIZE);
```

Reply:
@altmannmarcelo it is better to avoid this rewriting header version. start_LSN is OK. I would rather suppress check that does empty redo check.

Reply:
I have suppressed checks in 8.0.30 which are not valid for pre 8.0.30 files like
1. max file size to be 4G 
2. min start_lsn to be > LOG_START_LSN (8.0.29 and older wrotes 0xc which is lesser than 2048(LOG_START_LSN)